components/script/dom/bindings/str.rs

```diff
@@ -40,6 +40,11 @@ impl ByteString {
         self.0.len()
     }
 
+    /// Checks if the ByteString is empty.
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
```

Comment:
The point of `is_empty` is that it can be faster than `len()`. In this case `Vec.is_empty()` does use `Vec.len()`, but that's an implementation detail that we shouldn't rely on. So this should be

```suggestion
        self.0.is_empty()
```